lib/models/endpoint.js

```diff
@@ -127,7 +128,7 @@ function purifyResponse(me, incoming) {
       outgoing.push(pruneUndefined({
         headers: purifyHeaders(response.headers),
         status: parseInt(response.status) || 200,
-        latency: parseInt(response.latency) || undefined,
+        latency: parseInt(response.latency || cli.getArgs().latency) || undefined,
```

Comment:
We'll need to do a bit more than this for cases where `new Stubby().start(options)` is used, as it is not always started from a command-line invocation.